tests/Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cf
 RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
     ln -s /cache/images /build/images && \
     ln -s /cache/images /build/virt-builder && \
+    ln -s /cache/images /build/libvirt/qemu/log && \
```

Comment:
Are you sure that you don't want to put them into the /cache/images/log/ subdirectory? Seems nicer to keep the images and qemu logs apart. Not a blocker of course, just a question.